src/state/slices/opportunitiesSlice/constants.ts

```diff
@@ -35,11 +36,18 @@ export const foxEthStakingContractAddresses = [
 
 export type FoxEthStakingContractAddress = typeof foxEthStakingContractAddresses[number]
 
-export const isFoxEthStakingContractAddress = (
-  address: string,
+const isFoxEthStakingContractAddress = (
+  address: FoxEthStakingContractAddress | string,
 ): address is FoxEthStakingContractAddress =>
   foxEthStakingContractAddresses.includes(address as FoxEthStakingContractAddress)
 
+export const assertIsFoxEthStakingContractAddress: (
+  value: FoxEthStakingContractAddress | string,
+) => asserts value is FoxEthStakingContractAddress = getTypeGuardAssertion(
+  isFoxEthStakingContractAddress,
+  "assertIsFoxEthStakingContractAddress: Contract address isn't a known ETH/FOX staking address",
```

Comment:
That should already be part of the call stack? If so, I'd remove it to keep the error simple but fine if we don't

Reply:
https://github.com/shapeshift/web/pull/3485/commits/a450f073ae08b363efca2aa6e4decac3c6558fa9